x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/user/ProfileHasPrivilegesRequest.java

```diff
@@ -36,7 +36,7 @@ public class ProfileHasPrivilegesRequest extends ActionRequest {
         PARSER.declareStringArray(constructorArg(), Fields.UIDS);
         PARSER.declareField(
             constructorArg(),
-            parser -> RoleDescriptor.parsePrivilegesToCheck("profile_has_privileges_request", parser),
+            parser -> RoleDescriptor.parsePrivilegesToCheck("profile_has_privileges_request", false, parser),
```

Comment:
I don't think so, because there's nothing downstream relying on the value to have a certain value (ie part of the reason to make it a class member was to not code assumptions like that).